### ISSUE TITLE
Accessibility: WCAG 2.1 Level A: 1.3.1 Info and Relationships

### ISSUE THREAD

**Issue by rnunez80** [eea/volto-tabs-block#83]
WCAG 2.1 Level A
Ensures elements with an ARIA role that require parent roles are contained by them
Impact:  critical
Violations WCAG 2.1 Level A: 1.3.1 Info and Relationships
Element Location
.styled-tab > div[role="tab"]

Element Source
<div tabindex="0" role="tab">

Failure Summary
Fix any of the following: Required ARIA parent role not present: tablist

**Comment by ichim-david** [comment]
@rnunez80 Thank you for filling this issue, this was fixed many months ago, please try with the latest version, and if you can still trigger this than please re-open and show which tool was used for testing and what options you were using and taking a screenshot with the tab used.
